storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -466,6 +496,14 @@ public int remoteLogManagerThreadPoolSize() {
         return remoteLogManagerThreadPoolSize;
     }
 
+    public int remoteLogManagerCopierThreadPoolSize() {
+        return remoteLogManagerCopierThreadPoolSize;
+    }
+
+    public int remoteLogManagerExpirationThreadPoolSize() {
+        return remoteLogManagerExpirationThreadPoolSize;
+    }
+
```

Comment:
Should we also update the equals and hashcode?

Reply:
@kamalcph yes, I think we should. I will create a PR for it.

Reply:
@kamalcph here is the PR for it https://github.com/apache/kafka/pull/16173